properties.schema

```diff
@@ -38,6 +38,15 @@
                   "validators": ["required"],
                   "help": "Allows you to set whether Bookmarking is done at page, block or component level."
                 },
+                "_showPrompt": {
+                  "type": "boolean",
+                  "required": true,
+                  "default": true,
+                  "title": "Show prompt",
+                  "inputType": "Checkbox",
+                  "validators": [],
+                  "help": "Controls whether the Bookmarking prompt is enabled or disabled. False returns the user to their bookmarked position automatically."
```

Comment:
Second sentence might be more clear with something like:

_Uncheck to return the user to their..._

@moloko?

Reply:
Or "If not enabled, the user will be returned to their bookmarked position automatically" - that way you avoid referring to any UI control.

Reply:
Thanks for the suggestions @tomgreenfield @moloko. Wording changed.